game/services.py

```
@@ -242,4 +244,125 @@ def check_puzzle_achievements(user, stats):
+
+        award_font = ImageFont.truetype(
+            "C:/Windows/Fonts/georgiab.ttf",
+            32
+        )
+
+        name_font = ImageFont.truetype(
+            "C:/Windows/Fonts/georgiai.ttf",
+            60
+        )
+
+    except Exception:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)